edu.gemini.lch.services/src/main/scala/edu/gemini/lch/services/impl/JSpOCCredentials.scala

```diff
@@ -0,0 +1,18 @@
+package edu.gemini.lch.services.impl
+
+import edu.gemini.lch.model.Site
+import org.apache.commons.httpclient.UsernamePasswordCredentials
+
+import scala.collection.immutable.HashMap
+
+object JSpOCCredentials {
+  val gsUserName = System.getenv("GS_USERNAME")
```

Comment:
It might be worth making this a little safer so we don't have `null`s floating around.

```scala
def unsafeGetEnv(key: String): String =
  sys.env.getOrElse(key, sys.error(s"Missing environment variable $key"))

val gsUserName = unsafeGetEnv("GS_USERNAME")
// etc
```